snuba/cli/cleanup.py

```diff
@@ -13,13 +14,16 @@
               help="If true, only print which partitions would be dropped.")
 @click.option('--database', default='default',
               help='Name of the database to target.')
-@click.option('--table', default=settings.DEFAULT_LOCAL_TABLE,
-              help='Name of the table to target.')
+@click.option('--dataset', default='events', type=click.Choice(['events']),
+              help='The dataset to target')
 @click.option('--log-level', default=settings.LOG_LEVEL, help='Logging level to use.')
-def cleanup(clickhouse_server, dry_run, database, table, log_level):
+def cleanup(clickhouse_server, dry_run, database, dataset, log_level):
     from snuba.cleanup import run_cleanup, logger
     from snuba.clickhouse import ClickhousePool
 
+    dataset = get_dataset(dataset)
+    table = dataset.SCHEMA.get_table_name()
```

Comment:
Shouldn't this explicitly pin to the local table?

Reply:
fixed